source/collaborate/extend-mattermost-with-integrations.rst

```diff
@@ -1,30 +1,24 @@
 
-- `GitHub <https://github.com/mattermost/mattermost-plugin-github/blob/master/README.md#installation>`__
-- `GitLab <https://github.com/mattermost/mattermost-plugin-gitlab#readme>`__
-- `Jira <https://github.com/mattermost/mattermost-plugin-jira#readme>`__
-- `ServiceNow <https://github.com/mattermost/mattermost-plugin-servicenow>`__
-- `Zoom <https://github.com/mattermost/mattermost-plugin-zoom/blob/master/README.md#installation>`__
+- :ref:`AI Copilot <path/page>`
+- :ref:`Export Mattermost channel data <integrate/export-mattermost-channel-data:usage>`
+- :ref:`GitHub <integrate/github-interoperability:usage>`
+- :ref:`GitLab <integrate/gitlab-interoperability:usage>`
+- :ref:`Jira <integrate/jira-interoperability:usage>`
+- :ref:`Microsoft Teams <integrate/ms-teams-interoperability:usage>`
+- :ref:`Monitor performance metrics <integrate/monitor-performance-metrics:usage>`
+- :ref:`Perform legal holds <integrate/perform-legal-holds:usage>`
+- :ref:`ServiceNow <integrate/servicenow-interoperability:usage>`
+- :ref:`Zoom <integrate/zoom-interoperability:usage>`
 
-Latest integrations
--------------------
+.. tip::
 
-.. toctree::
-   :maxdepth: 1
-   :hidden:
-
-   Collaborate within connected MS Teams </collaborate/collaborate-within-connected-microsoft-teams>
-   Collaborate within embedded MS Teams </collaborate/collaborate-within-embedded-microsoft-teams>
-   Use the Mattermost Google Calendar plugin </collaborate/use-mattermost-google-calendar-plugin>
-
-* :doc:`Collaborate within a Microsoft Teams instance </collaborate/collaborate-within-connected-microsoft-teams>` - Learn how to collaborate with colleagues when Mattermost is connected to Microsoft Teams.
-* :doc:`Collaborate within an embedded Microsoft Teams instance </collaborate/collaborate-within-embedded-microsoft-teams>` - Learn how to collaborate with colleagues when Mattermost is embedded within Microsoft Teams.
-* :doc:`Use the Mattermost Google Calendar plugin </collaborate/use-mattermost-google-calendar-plugin>` - Learn how to manage events using a two-way integration between Mattermost and Google Calendar.
\ No newline at end of file
+   `Visit the Mattermost Marketplace <https://mattermost.com/marketplace/>`__ to find open source, community-supported integrations to common developer tools like `CircleCI <https://mattermost.com/marketplace/circleci/>`__, `Opsgenie <https://mattermost.com/marketplace/opsgenie/>`__, `PagerDuty Notifier <https://mattermost.com/marketplace/pagerduty/>`__, and `Zendesk App <https://mattermost.com/marketplace/zendesk-app/>`__; productivity tools like `Autolink <https://mattermost.com/marketplace/autolink-plugin/>`__, `ToDo <https://mattermost.com/marketplace/todo/>`__, and `WelcomeBot <https://mattermost.com/marketplace/welcomebot-plugin/>`__; as well as social tools like `Memes <https://mattermost.com/marketplace/memes-plugin/>`__ and `GIFs <https://mattermost.com/marketplace/giphy-plugin/>`__ that are freely available for use and customization.
```

Comment:
I don't think it's wise to advertise the Zendesk App, given that we never used it in production and we are not sure it works.

Reply:
Thank you! I'll remove.